systems/plants/@Manipulator/Manipulator.m

```diff
@@ -36,6 +36,73 @@
       C_times_v = C-G;
     end
     
+    function [xdot,dxdot] = Hdynamics(obj,t,x,u)
```

Comment:
Why was it necessary to add that function?



On Thu, Jul 30, 2015 at 10:54 PM, Twan Koolen <notifications@github.com>
wrote:

> In systems/plants/@Manipulator/Manipulator.m
> <https://github.com/RobotLocomotion/drake/pull/1212#discussion_r35943184>:
>
> > @@ -36,6 +36,73 @@
> >        C_times_v = C-G;
> >      end
> >
> > +    function [xdot,dxdot] = Hdynamics(obj,t,x,u)
>
> Why was it necessary to add this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/RobotLocomotion/drake/pull/1212/files#r35943184>.
>